tests/src/clutter/mews.ts

```diff
@@ -122,6 +122,18 @@ export default (orchestrator: Orchestrator<any>) =>
     mews = await bob.call("mews", "mews_feed", { option: "" });
     t.equal(mews.length, 0);
 
+    try {
+      await bob.call("mews", "follow", bobPubKey);
+      await sleep(777);
```

Comment:
This and the following line are never reached, because `bob.call(...)` throws and error and the error is caught.

Reply:
I removed the sleep line, but the t.fail() is still needed since if it doesn't throw an error the test should fail.

Reply:
Ah, true 👍